templates/openwisp2/settings.py

```diff
@@ -368,3 +384,17 @@
 {{ instruction }}
 
 {% endfor %}
+
+{% if openwisp2_monitoring %}
+TIMESERIES_DATABASE = {
+    'BACKEND': '{{ openwisp2_timeseries_database.backend }}',
+    'USER': '{{ openwisp2_timeseries_database.user }}',
+    'PASSWORD': '{{ openwisp2_timeseries_database.password }}',
+    'NAME': '{{ openwisp2_timeseries_database.name }}',
+    'HOST': '{{ influxdb_http_ip }}',
+    'PORT': '{{ influxdb_http_port }}',
+}
+
```

Comment:
@Purhan move this block above https://github.com/openwisp/ansible-openwisp2/pull/262/files#diff-8c46e786c6289dcbdc2b3e43782ff1cd1308bf2cde778ac89485ea48da22095bL362-L368

Reply:
above what?

Reply:
https://github.com/openwisp/ansible-openwisp2/blob/f05a3b79e537482a513aa0ac40914d22c4a2f228/templates/openwisp2/settings.py#L378-L386

Reply:
Don (https://github.com/openwisp/ansible-openwisp2/pull/262/commits/6d01973916f78d9f92763b17e77f2bf9c2bb5509) :+1: